### PR TITLE
update to babel 6

### PR DESCRIPTION
**Pull request by BrandonOCasey**
* re-generated README table of contents
* updated babel to babel 6

### CODE REVIEW

**Comment by misteroneill** [comment]
Nice, I'll tinker with this over the weekend or early next week!

File: src/app/templates/_.babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015-loose"],
```
Comment (gkatsev) [inline src/app/templates/_.babelrc]: we probably should default to non-loose and maybe have an option for loose mode? Plugins that don't care or need IE8 don't want loose mode. Also, for IE8 support, we want the other two or three transforms from videojs.

Comment (misteroneill) [inline src/app/templates/_.babelrc]: Perhaps a "Do you need IE8 support?" prompt is in order...

Comment (misteroneill) [inline src/app/templates/_.babelrc]: It also appears that [babel-preset-es2015-loose](https://www.npmjs.com/package/babel-preset-es2015-loose) is deprecated and now built into [babel-preset-es2015](https://www.npmjs.com/package/babel-preset-es2015) as an option - `{loose: true}`.

Comment (BrandonOCasey) [inline src/app/templates/_.babelrc]: nice call, must have just come out

**Comment by BrandonOCasey** [comment]
@Pat @gkatsev added the "support ie8 option" and tests for it

File: src/test/app.test.js [hunk 2]
```diff
@@ -4,9 +4,9 @@ import _ from 'lodash';
 import fs from 'fs-extra';
 import path from 'path';
 import {assert, test as helpers} from 'yeoman-generator';
-
 import * as libs from './libs';
 import packageJSON from '../generators/app/package-json';
+import {assert as chaiAssert} from 'chai';
```
Comment (misteroneill) [inline src/test/app.test.js]: Is there a reason chai is needed here vs using the assert module provided by yeoman? It extends Node's [assert](https://nodejs.org/api/assert.html) module; so, you should have the methods you need available.

File: src/app/index.js [hunk 3]
```diff
@@ -228,6 +229,11 @@ export default yeoman.generators.Base.extend({
       default: defaults.sass
     }, {
       type: 'confirm',
+      name: 'ie8',
+      message: 'Do you want to support ie8?',
```
Comment (misteroneill) [inline src/app/index.js]: Minor point, but I think "ie" should be capitalized here.

File: src/app/templates/_.babelrc [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [["es2015", {"loose": true}]],
+<% if (ie8) { -%>
+	"plugins": [
```
Comment (misteroneill) [inline src/app/templates/_.babelrc]: Indenting looks off in this block, maybe?

File: src/app/index.js (hunk 3 above)
Comment (BrandonOCasey) [inline src/app/index.js]: Yeah, may even want to spell it out just in case good call

File: src/test/app.test.js (hunk 2 above)
Comment (BrandonOCasey) [inline src/test/app.test.js]: I just looked at the yeoman api docs and didn't see anything, but I never realized that it inherits from node assert. I will look into it

**Comment by misteroneill** [comment]
LGTM

**Pull request merged**